### ISSUE TITLE
Unable to convert oni2avi using Windows

### ISSUE THREAD

**Issue by MritulaC** [KirillLykov/oni2avi#13]
Hi, I am trying to convert oni files to avi files using windows. Im able to view the display of .oni files but not able to save the output files. Can you kindly help.

I got the following message 
Boost version: 1.62.0
-- Configuring done
-- Generating done
-- Build files have been written to: C:/....../oni2avi

 The error
C:\oni2avi>/oni2avi G:/oni_files/1.oni G:/oni_files/1.avi --codec=FLV1

C:\Users\Harit\oni2avi> /oni2avi G:/DGAIT/oni_files/1.oni G:/DGAIT/oni_files/1.avi --codec=FLV1
'./oni2avi' is not recognized as an internal or external command,
operable program or batch file.

Kindly let me know how to overcome this.

Thanks,
 Mritula

**Comment by KirillLykov** [comment]
I'm not sure about windows, just try without `./`, just `oni2avi G:...`

**Comment by MritulaC** [comment]
Hi KirillLykov,
 Thanks for your response. I did try that. Im able to play/view the content of the .oni file but unable to save it or its not getting saved to the output file. FYI I used CMAKE. Build was successful..
C:\users\oni2avi> G:\video1.oni  G:\video1.avi --codec=FLV1

**Comment by KirillLykov** [comment]
So the oni is valid but avi was not created by `oni2avi`, am I right?
You can also try to specify options  explicitly: `oni2avi --input-file=G:\video1.oni --output-file=G:\video1.avi  --codec=FLV1`

**Comment by MritulaC** [comment]
Still not working with windows.
With MAC I am getting the following error
/github/oni2avi/build$ make
[ 50%] Building CXX object CMakeFiles/oni2avi.dir/oni2avi.cpp.o
/home/usr1/github/oni2avi/oni2avi.cpp:21:20: fatal error: OpenNI.h: No such file or directory
compilation terminated.
CMakeFiles/oni2avi.dir/build.make:62: recipe for target 'CMakeFiles/oni2avi.dir/oni2avi.cpp.o' failed
make[2]: *** [CMakeFiles/oni2avi.dir/oni2avi.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/oni2avi.dir/all' failed
make[1]: *** [CMakeFiles/oni2avi.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by MritulaC** [comment]
Just wanted to confirm with you, The below statement needs to be executed from Command Prompt right?
You can also try to specify options explicitly: 

oni2avi --input-file=G:\video1.oni --output-file=G:\video1.avi --codec=FLV1 <- This statement?

**Comment by MritulaC** [comment]
Boost is the main problem. Even in MAC the errors are created only because of BOOST I suppose. Can you help us with this?

**Comment by KirillLykov** [comment]
Right, in the command line. For mac, the problem is OpenNI - check that OpenNI2 has been installed and that path to include and libraries are correct in the Makefile (check lines 22-23). I propose to use Makefile instead of cmake for mac, since it is what I used. Currently, I suppose that this library is installed into `/usr/local/include/ni2/` which is where it is installed if you use homebrew.

**Comment by MritulaC** [comment]
Hi,
 Im sorry I think the problem is with oni2avi.exe is not getting created. The CMAKEOUTPUT.log  shows only this 
Done Building Project "C:\oni2avi\build\CMakeFiles\3.11.0\CompilerIdC\CompilerIdC.vcxproj" (default targets).

Build succeeded.
    0 Warning(s)
    0 Error(s)

but not as oni2avi.exe

Can you kindly suggest me. Its been a week im struck with this

**Comment by KirillLykov** [comment]
well, I haven't used cmake on windows but since it created a project, may be you can open it with Visual Studio and compile there?
